modules/UI/UI.js

```diff
@@ -305,45 +297,6 @@ UI.toggleFilmstrip = function() {
  */
 UI.toggleChat = () => APP.store.dispatch(toggleChat());
 
-/**
- * Handle new user display name.
- */
-UI.inputDisplayNameHandler = function(newDisplayName) {
-    eventEmitter.emit(UIEvents.NICKNAME_CHANGED, newDisplayName);
-};
-
-// FIXME check if someone user this
-UI.showLoginPopup = function(callback) {
-    logger.log('password is required');
-
-    const message
-        = `<input name="username" type="text"
-                placeholder="user@domain.net"
-                data-i18n="[placeholder]dialog.user"
-                class="input-control" autofocus>
-         <input name="password" type="password"
-                data-i18n="[placeholder]dialog.userPassword"
-                class="input-control"
-                placeholder="user password">`
-
-    ;
```

Comment:
Have you removed this? As people are using it and just submitted a PR fixing it https://github.com/jitsi/jitsi-meet/pull/8233

Reply:
We agreed offline that this is not used anymore.